source/Engine.cpp

```diff
@@ -942,8 +942,23 @@ void Engine::Draw() const
 		if(messagePoint.Y() < messageBox.Top())
 			break;
 		float alpha = (it->step + 1000 - step) * .001f;
-		Color color(alpha, 0.f);
-		messageLine.Draw(messagePoint, color);
+		const Color* color;
```

Comment:
Always initialize pointers:

```suggestion
		const Color *color = nullptr;
```

Reply:
Done.